proto/stride/stakeibc/trade_route.proto

```diff
@@ -30,46 +29,58 @@ message TradeRoute {
   // ibc denom of the reward on the trade chain, input to the swap
   string reward_denom_on_trade_zone = 3;
   // ibc of the host denom on the trade chain, output from the swap
-  string target_denom_on_trade_zone = 4;
+  string host_denom_on_trade_zone = 4;
   // should be the same as the native host denom on the host chain
-  string target_denom_on_host_zone = 5;
+  string host_denom_on_host_zone = 5;
 
-  TradeHop host_to_reward_hop = 6 [ (gogoproto.nullable) = false ];
-  TradeHop reward_to_trade_hop = 7 [ (gogoproto.nullable) = false ];
-  TradeHop trade_to_host_hop = 8 [ (gogoproto.nullable) = false ];
+  // ICAAccount on the host zone with the reward tokens
+  // This is the same as the host zone withdrawal ICA account
+  ICAAccount host_account = 6 [ (gogoproto.nullable) = false ];
+  // ICAAccount on the reward zone that is acts as the intermediate
+  // receiver of the transfer from host zone to trade zone
+  ICAAccount unwind_account = 7 [ (gogoproto.nullable) = false ];
```

Comment:
hate to bring this up, but for consistency I think this should be called the reward_account
The host_account and trade_account are named after their chains since they are the relevant accounts in the context of this trade route on those chains.
unwind is the type for the ica account in the same way the host_account has  type of withdrawal.  There could be more than one ica on a given chain which is why naming by type makes sense in some contexts like on the hostZone struct but for consistency here I think we would want to name this reward_account

Reply:
I tried to tell em, but they just wouldn't listen...

kidding haha, but agreed, I'll make the change